dashboard/app/models/script.rb

```diff
@@ -401,71 +401,82 @@ def self.add_script(options, data)
     levels_by_key = script.levels.index_by(&:key)
 
     # Overwrites current script levels
-    script.script_levels = data.map do |row|
-      row.symbolize_keys!
+    script.script_levels = raw_script_levels.map do |raw_script_level|
+      raw_script_level.symbolize_keys!
 
-      # Concepts are comma-separated, indexed by name
-      row[:concept_ids] = (concepts = row.delete(:concepts)) && concepts.split(',').map(&:strip).map do |concept_name|
-        (Concept.by_name(concept_name) || raise("missing concept '#{concept_name}'"))
-      end
+      assessment = false
```

Comment:
Hmm, I saw that it was a boolean so I figured it should default to false, but it looks like the existing behavior sets it to `true` or `nil`. Switched.